scripts/monitor.sh

```diff
@@ -4,12 +4,13 @@
 BASE_DIR="/home/potato/NFC_Tracking"
 
 # Define the names and paths of the C programs to be monitored.
-PROGRAM_NAME_1="read_ultralight"
-PROGRAM_NAME_2="button_listener"
-PROGRAM_NAME_3="screen.py"
-PROGRAM_PATH_1="$BASE_DIR/$PROGRAM_NAME_1"
-PROGRAM_PATH_2="$BASE_DIR/$PROGRAM_NAME_2"
-PROGRAM_PATH_3="$BASE_DIR/$PROGRAM_NAME_3"
+PROGRAM_NAMES=(
+    "nfc_tap_listener.c"
+    "operation_tap_listener.c"
+    "screen.py"
```

Comment:
Owner, so I can't make a suggestion directly, but here is my suggestion:
`"display_and_data_handler.py"`